polars/polars-core/src/frame/asof_join/groups.rs

```diff
@@ -969,4 +1018,65 @@ mod test {
 
         Ok(())
     }
+
+    #[test]
```

Comment:
This test fails because of your invalid comparison with `Vec`. 

We also want most tests on the python side. So probably better to move this test there.

Reply:
Duh. I already have this test in the Python side so I simply removed this one.